### PR TITLE
[feat]: Added menu for cards

### PR DESCRIPTION
**Pull request by lannisterrr**
Added - Menu card component
Added - Custom hook to handle addEventListeners.

### CODE REVIEW

File: src/components/ListingVideoComponent.js [hunk 1]
```diff
@@ -19,13 +24,20 @@ function ListingVideoComponent({ video }) {
...
           <p className="f-5 p-1 f-bold">{title}</p>
-          <button className="lisiting-video__info-button">
+          <button
+            onClick={() => {
+              setShowMenu(showMenu => !showMenu);
+            }}
+            className="lisiting-video__info-button"
+          >
             <i class="fa-solid fa-ellipsis-vertical f-8 f-bold t-c-1"></i>
           </button>
```
Comment (ashishnnnnn) [inline src/components/ListingVideoComponent.js:38]: replace class with className ..

File: src/components/VideoMenu.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        onClick={() => console.log('added to watch later')}
+        className="f-6 w-100 menu-item pointer"
+      >
+        <i class="fa-regular fa-clock f-8 p-h-2"></i>
+        Add to watch Later
+      </span>
+
+      <span className="f-6 w-100 menu-item pointer">
+        <i class="fa-regular fa-folder f-8 p-h-2"></i>
+        Add to Playlist
```
Comment (ashishnnnnn) [inline src/components/VideoMenu.js:14]: Same ...

**Review by ashishnnnnn — APPROVED** [review]
Overall code seems good..
Just Do those changes..

**Review by RakshithBhat03 — APPROVED** [review]
The code looks clean and well structured. Good use of custom hooks.

**Pull request merged**